package.json

```diff
@@ -66,6 +66,7 @@
     "@fortawesome/vue-fontawesome": "^2.0.8",
     "@google/model-viewer": "^1.12.1",
     "@keeex/qrcodejs-kx": "^1.0.2",
+    "@kodadot/ui": "workspace:*",
```

Comment:
thinking to have cooler name for that 🤔 may @exezbcz/ @yangwao have some ideas?

Reply:
something which is more contextual yes